ffi/build.py

```diff
@@ -109,7 +109,7 @@ def main_posix(kind, library_ext):
 
     out = out.decode('latin1')
     print(out)
-    if not (out.startswith('8.0.') or out.startswith('7.0.')
+    if not (out.startswith('9.0.') or out.startswith('7.0.')
```

Comment:
```suggestion
    if not (out.startswith('9.0.') or out.startswith('8.0.') or out.startswith('7.0.')
```

This will drop support for 8.0 which I assume was not intended.